backend/substrapp/tasks/utils.py

```diff
@@ -220,6 +220,87 @@ def list_files(startpath):
         logger.info(f'{startpath} does not exist.')
 
 
+def build_docker_image(dockerfile_path, image_name):
+    dockerfile_fullpath = os.path.join(dockerfile_path, 'Dockerfile')
+    current_namespace = os.getenv('NAMESPACE')
+    docker_cache = os.getenv('DOCKER_CACHE_PVC')
+
+    config.load_incluster_config()
+
+    batch_v1 = client.BatchV1Api()
+
+    container = client.V1Container(
+        name="kaniko",
+        image="gcr.io/kaniko-project/executor:latest",
+        args=[
+            f"--dockerfile={dockerfile_fullpath}",
+            f"--context=dir://{dockerfile_path}",
+            f"--destination={os.getenv('REGISTRY')}:5000/{image_name}",
+            "--cache=true",
+            "--insecure"
+        ],
+        volume_mounts=[
+            {'name': 'dockerfile', 'mountPath': f"{os.getenv('MEDIA_ROOT')}subtuple", 'readOnly': True},
```

Comment:
It may be better to build the mountPath with os.path.join
Moreover you can certified that MEDIA_ROOT has a '/' at the end with 
```python
# Get real path and ensure there is a suffix / at the end of the path
os.path.join(os.path.realpath(to_directory), '')
```